### PR TITLE
Allow response effects html to be an html node

### PR DESCRIPTION
**Pull request by godismyjudge95**
1️⃣ Is this something that is wanted/needed? Did you create an issue / discussion about it first?
It is needed to be able to properly fix Livewire's integration with Vue 3 (separate pull request coming soon on the livewire/vue repo).

2️⃣ Does it contain multiple, unrelated changes? Please separate the PRs out.
No, just a single change.

3️⃣ Does it include tests, if possible? (Not a deal-breaker, just a nice-to-have)
Sorry no, but all included tests should still work.

4️⃣ Please include a thorough description of the improvement and reasons why it's useful.
I added a simple condition to check that the `response.effects.html` is a string before running `trim()` on it.  This prevents an error if the `response.effects.html` is an html node - which can happen when the response is hooked into via the `message.received` hook.

Basically, this code should function the same in all use cases except when the `message.received` hook is used to change it to an html node.

5️⃣ Thanks for contributing! 🙌
Thank you for such a great library.  This is my first pull request to Livewire, hopefully many more in the future.

### CODE REVIEW

**Comment by godismyjudge95** [comment]
Re-opened this due to a more solid example on why it is needed:  https://github.com/livewire/livewire/pull/4127

**Comment by calebporzio** [comment]
Hey @godismyjudge95, I just commented on the livewire/vue PR.

My main hesitancy with adding this to core is now the future of Livewire is locked into allowing dom nodes as morph targets, which currently that isn't a constraint we have to honor.

However, if it is the only way to allow you're (hopefully) third-party Vue 3 support plugin I'm willing to consider.

Let me know if this is the only way and I'll seriously consider merging this. Thanks!
